jsonpb/jsonpb_test.go

```diff
@@ -473,6 +476,33 @@ func TestMarshalingNil(t *testing.T) {
 	}
 }
 
+func TestMarshalIllegalTime(t *testing.T) {
+	tests := []struct {
+		pb      proto.Message
+		errWant error
```

Comment:
I don't think we care that much about the exact error message. Let's just change this to a "fail bool" that checks whether it passes or not.

Reply:
Done.